src/models/onnxruntime_api.h

```diff
@@ -93,11 +94,14 @@ p_session_->Run(nullptr, input_names, inputs, std::size(inputs), output_names, o
 #define PATH_MAX (4096)
 #endif
 
-#define LOG_DEBUG(...) Generators::Log("debug", __VA_ARGS__)
-#define LOG_INFO(...) Generators::Log("info", __VA_ARGS__)
-#define LOG_WARN(...) Generators::Log("warning", __VA_ARGS__)
-#define LOG_ERROR(...) Generators::Log("error", __VA_ARGS__)
-#define LOG_FATAL(...) Generators::Log("fatal", __VA_ARGS__)
+#define LOG_WHEN_ENABLED(LOG_FUNC) \
```

Comment:
How about move all these macros to `logging.h`, including the androids and other ones the I added in my PRs. That would make this file less bloated.

Reply:
Let's avoid using macros as a general-purpose logging utility for now. If it comes to that, we can make this happen later.

Reply:
sure, no problem